src/base/base.scss

```diff
@@ -15,7 +15,7 @@ button {
 body {
   font-family: $body-font;
   font-size: $body-font-size;
-  line-height: $body-line-height;
+  line-height: var(--primer-text-body-lineHeight-medium, $body-line-height);
```

Comment:
@vdepizzol I'm a bit concerned about this one. It seems like _most_ typography settings actually just fallback to this line-height on the body. I did set explicit line-height values on all the headings in PCSS, but there might be more in dotcom that we don't see reflected here. Any thoughts?

Reply:
@langermank I think it makes sense to keep the `body` `line-height` at `1.5`, and then apply the `body-lineHeight-medium` token to any component that requires the body-medium typography style. Wdyt?

Reply:
Sounds good to me! I wonder if we should add `font-size` and `line-height` to the `p` tag in `typography-base` 🤔